auraescript/src/bin/main.rs

```diff
@@ -65,8 +65,8 @@ fn main() -> anyhow::Result<()> {
     let main_module = resolve_path(&args[1].clone(), current_dir()?.as_path())?;
     let mut main_worker = init(main_module.clone());
 
-    let _ =
-        main_worker.execute_script("", "Deno.core.initializeAsyncOps();")?;
+    let _ = main_worker
+        .execute_script("", "Deno.core = Deno[Deno.internal].core;")?;
```

Comment:
I assume deno put that into an internal namespace on purpose. is exposing it safe?

Reply:
I believe so, I think they mostly wanted to add _some_ level of privacy, but it is still always technically accessible via `Deno[Deno.internal]`.

The reason why I sent it "up a level" was just to minimize the changes that needed to be made to existing TypeScript files and TypeScript codegen. But I could also see an argument for just using `Deno[Deno.internal]` in the TypeScript code because someone who is used to seeing regular Deno code might be confused and would have to find this code to understand why `Deno.core` works.

Open to either option.

Reply:
what you've done is fine, it's just a little odd that they (deno) decided to put expected public functionality into an internal namespace.